tests/pet.test.js

```diff
@@ -0,0 +1,112 @@
+const Pet = require('../src/pet');
+
+describe('create a new object with a set of properties', () => {
+  let pet;
+  beforeEach(() => {
+    pet = new Pet('Boris');
+  });
+
+    test('returns an object', () => {
+      expect(new Pet('Boris')).toBeInstanceOf(Object);
+    });
+
+    test('sets the name property', () => {
+      expect(pet.name).toEqual('Boris');
+    });
+
+    test('initial age of 0', () => {
+      expect(pet.age).toEqual(0);
+    });
+
+    test('increments the age by 1', () => {
+      pet.growUp();
+      expect(pet.age).toEqual(1);
+    });
+
+    test('initial hunger of 0', () => {
+      expect(pet.hunger).toEqual(0);
+    });
+
+    test('increments the hunger by 5', () => {
+      pet.growUp();
+      expect(pet.hunger).toEqual(5);
+    });
+
+    test('increments the fitness by 4', () => {
+      pet.fitness = 4;
+      pet.walk();
+      expect(pet.fitness).toEqual(8);
+    });
+
+    test('increases fitness by to a maximum of 10', () => {
+      pet.fitness = 8;
+      pet.walk();
+      expect(pet.fitness).toEqual(10);
+    });
+
+    test('decreases hunger level by 3 to a minimum of 0', () => {
```

Comment:
add one more test for decreasing hunger level by 3 on it's own like you did with fitness :)